src/appMain/routes/users/routes/UserDetail/components/KubeconfigValidity.js

```diff
@@ -15,7 +15,13 @@ import { capitalizeFirstLetter } from "../../../../../../utils";
 const KubeconfigValidity = ({ settings, onSave, orgSetting, orgId }) => {
   if (!settings) return null;
   const dispatch = useDispatch();
+  const min_minutes = 10;
+  const max_minutes = 30 * 24 * 60;
```

Comment:
let's not have the same max validations for kubeconfig

Reply:
also, kubeconfig validity was in hours earlier ? why have we modified this to minutes ?

Reply:
Yes, it was in hours. We changed this so that users would not be able to enter time in decimals. e.g. 1/2 hr or 1 hr 45 mins etc. Hence we decided to go for minutes here as well. By default this value is set to equivalent of 8 hours in minutes

Reply:
reverted. No validation exists for kubeconfig validity as of now.